cartogram_generator/fill_with_density.c

```diff
@@ -303,7 +303,7 @@ BOOLEAN fill_with_density1 (char *map_file_name, char *area_file_name,
     r = 245.0;  
     g = 235.0;
     b = 179.0;
-    sscanf(line, "%d%*[, ]%lf%*[, a-zA-Z]%lf%*[, ]%lf%*[, ]%lf", &id, &area, &r, &g, &b);
+    sscanf(line, "%d%*[, ]%lf%*[, ]%*[ 0-9a-zA-Z]%*[, ]%lf%*[, ]%lf%*[, ]%lf", &id, &area, &r, &g, &b);
```

Comment:
the scanf formatting didn't permit region names that were numbers. It would treat them as the red color.

Reply:
Thanks, the previous version indeed misinterpreted numeric IDs